pennylane/_qubit_device.py

```diff
@@ -769,20 +773,23 @@ def statistics(
               ``shot_range=[35, 135]``, ``bin_size=100``.
         """
         if observables is not None:
-            if isinstance(observables, QuantumScript):
-                circuit = observables
-                measurements = circuit.measurements
-            else:
-                warnings.warn(
-                    message="Using a list of observables in ``QubitDevice.statistics`` is "
-                    "deprecated. Please use a ``QuantumTape`` instead.",
-                    category=UserWarning,
-                )
-                measurements = observables
-        elif circuit is not None:
+            warnings.warn(
+                message="The ``observables`` argument in ``QubitDevice.statistics`` is deprecated. "
+                "Please use ``circuit`` instead.",
+                category=UserWarning,
+            )
+            circuit = observables
+        elif circuit is None:
+            raise ValueError("Please provide a circuit into the statistics method.")
+        if isinstance(circuit, QuantumScript):
```

Comment:
When running:
```pycon
>>> dev.statistics(observables=[])
```
two warnings are raised. I would prefer if only the first warning is raised.

Reply:
It now only raises the first warning in that case :)